js/xapi-parent-forwarder.js

```diff
@@ -0,0 +1,18 @@
+// Forward xAPI events to parent window, making it possible to use the
+// following code from parent window to get events:
+// window.addEventListener('message', function (event) { <...> });
+H5P.externalDispatcher.on('xAPI', function (event) {
+  if (event.data && event.data.statement && event.data.statement.actor && event.data.statement.actor.mbox) {
+    if (H5PIntegration.user && H5PIntegration.user.mail_sha1sum) {
+      event.data.statement.actor = {
+        mbox_sha1sum: H5PIntegration.user.mail_sha1sum,
```

Comment:
@fnoks Greatly appreciated that the actor will be deleted here! Where's sha1 used though? I think it's pretty much broken and should not be used anymore.

Reply:
Hmm. mbox_sha1sum is part of the specification, and I didn't see any other options (https://xapi.com/blog/deep-dive-actor-agent/). Maybe we should just add a mail_sha2sum instead? For this (non-LRS) purpose, it probably doesn't matter much. What do you think?

Reply:
@fnoks Hmm, guess following what is there then is the best that you can do for now given that there's no extension option for the `actor` property.